packages/react-intersection-observer/README.md

```diff
@@ -13,13 +13,37 @@ $ yarn add @shopify/react-intersection-observer
 
 ## Usage
 
-This package exports an `IntersectionObserver` component. This component will create its own DOM node that will be observed, with optional `onIntersecting` and `onNotIntersecting` props being called as appropriate. `onIntersecting` and `onNotIntersecting` will be called with an `IntersectionObserverEntry` object, which describes the state of the intersection. The component also accepts a few additional props that correspond to the options used to construct an `IntersectionObserver`:
+### `useIntersection(options: Options)`
 
-- `threshold`: a number or array of number indicating the `intersectionRatio` that must be met before the observer is triggered
+The `useIntersection` hook takes in `IntersectionObserver` options, and returns a tuple of:
+
+- The most recent `IntersectionObserverEntry` value, which details the most recent state of the observer.
```

Comment:
This feels like a convoluted way to describe what it returns, maybe just `The state of the observer`?